src/plugins/messenger/MessengerPreview/components/MessengerImage/components/LightBoxHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRef } from "react";
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import CloseIcon from '../../../../../../webchat-ui/assets/baseline-close-24px.svg';
+import DownloadIcon from '../../../../../../webchat-ui/assets/baseline-download-24px.svg';
+
+export const getLightBoxHeader = ({ React, styled }: MessagePluginFactoryProps) => {
+
+    const Header = styled.div(({ theme }) => ({
+        position: 'absolute',
+        top: 0,
+        height: '56px',
+        width: '100%',
+        backgroundColor: theme.primaryColor,
+        overflow: 'hidden',
+        display: 'flex',
+        justifyContent: 'space-between'
+    }))
+
+    const Caption = styled.div({
+        display: 'flex',
+        alignItems: 'center',
+        marginLeft: '15px',
+        color: 'white',
+        fontWeight: '700',
+        fontSize: '16px'
+    })
+
+    const IconsGroup = styled.div({
+        display: 'flex',
+        alignItems: 'center',
+        marginRight: '10px'
+    })
+
+    const Icon = styled.button({
+        padding: '8px',
+        boxSizing: 'border-box',
+        backgroundColor: 'transparent',
+        border: 'none',
+        outline: 'none',
+        margin: 0,
+        transition: 'background-color .1s ease-out,color .1s ease-out,fill .1s ease-out',
+        color: 'white',
```

Comment:
Same here, we should use primaryContrastColor from these, so that it is visible on white backgrounds. Also same for 'fill' in line 47.